src/EnergyPlus/AirLoopHVACDOAS.cc

```diff
@@ -695,6 +697,21 @@ namespace AirLoopHVACDOAS {
                         ShowSevereError(state, format("Outlet node number is not found in {} = {}", CurrentModuleObject, CompName));
                         errorsFound = true;
                     }
+                    // Check node connection to ensure that the outlet node of the previous component is the inlet node of the current component
+                    if (CompNum > 1) {
+                        if (thisOutsideAirSys.InletNodeNum(CompNum) != thisOutsideAirSys.OutletNodeNum(CompNum - 1)) {
+                            ShowSevereError(state,
```

Comment:
The error message here and below should include the routinename and lead with the name of the DOAS System, e.g.:
```
getAirLoopMixer:  Node connection error in AirLoopHVAC:DedicatedOutdoorAirSystem=AirLoopHVAC DOAS.
Inlet node . . . .
```

Reply:
@mjwitte Done.